src/apps/material/login-redirect.test.ts

```diff
@@ -5,15 +5,20 @@ describe("Material", () => {
     window.sessionStorage.removeItem("user");
 
     cy.visit("/iframe.html?id=apps-material--default&type=bog&")
+      .getBySel("material-description")
+      .scrollIntoView()
```

Comment:
Consider if we can use `material-header-content` instead of `material-description`.

I am a bit surprised that we need to scroll to the description )which I think would be far down the page) when we want to use the reservation buttons which are placed in the header. 

Note that this occurs in many places throughout this PR.